### PR TITLE
add id translation

### PR DESCRIPTION
**Pull request by kiraware**
## Description
Support Bahasa Indonesia translation

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have run the `pre-commit run` command to format and lint.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] I have added my name and/or github handle to `AUTHORS.rst`
- [x] I have added my change to `CHANGES.rst`
- [ ] All new and existing tests passed.

### CODE REVIEW

File: simple_history/locale/id/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Indonesian translation for django-simple-history
+# Copyright (C) 2023
+# This file is distributed under the same license as the django-simple-history package.
+# Kira <kiraware@github.com>, 2023.
+#
+#, fuzzy
```
Comment (ddabble) [inline simple_history/locale/id/LC_MESSAGES/django.po:6]: Please remove this line 🙂 
```suggestion
```

File: simple_history/locale/id/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+"POT-Creation-Date: 2023-06-30 15:21+0800\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Kira <kiraware@github.com>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: id\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+#: .\simple_history\admin.py:102
```
Comment (ddabble) [inline simple_history/locale/id/LC_MESSAGES/django.po:20]: ...and add an empty line here:
```suggestion
"Plural-Forms: nplurals=1; plural=0;\n"

#: .\simple_history\admin.py:102
```

**Review by ddabble — CHANGES_REQUESTED** [review]
Looks good to me :) Just a couple minor requests:

**Comment by kiraware** [comment]
Done @ddabble. Thanks for your response.

**Review by ddabble — APPROVED** [review]
Thank you! 😊

**Comment by kiraware** [comment]
Your welcome :)

**Pull request merged**